R/branch.R

```diff
@@ -6,7 +6,7 @@
 #' @rdname branch
 #' @name branch
 #' @family git
-#' @inheritParams repository
+#' @inheritParams git_open
```

Comment:
If you support the basic premise of this PR, I will use a different technique to inherit the relevant piece of text everywhere.

Reply:
Relates to the `path` vs. `repo` comment below.